features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin_new/source/src/app/components/Throttling/Blacklist/List.jsx

```diff
@@ -0,0 +1,207 @@
+                    <ListItem button>
+                        <ListItemIcon>
+                            <DescriptionIcon />
+                        </ListItemIcon>
+                        <Link
+                            target='_blank'
+                            href={Configurations.app.docUrl
+            + 'learn/rate-limiting/blacklisting-whitelisting/#blacklisting-requests'}
+                        >
+                            <ListItemText primary={(
+                                <FormattedMessage
+                                    id='Throttling.Blacklist.Policy.List.help.link.one'
+                                    defaultMessage='Blacklisting requests'
+                                />
+                            )}
+                            />
+
+                        </Link>
+                    </ListItem>
+                </List>
+            </HelpBase>),
+        pageStyle: 'half',
+        title: intl.formatMessage({
+            id: 'Throttling.Blacklist.Policy.search.default',
+            defaultMessage: 'Blacklist Policies',
+        }),
+        EditTitle: intl.formatMessage({
+            id: 'Throttling.Blacklist.Policy.search.default',
+            defaultMessage: 'Blacklist Policies',
+        }),
+    };
+
+    const columProps = [
+        {
+            name: 'conditionId',
+            label: intl.formatMessage({
+                id: 'Admin.Throttling.Blacklist.Throttling.policy.table.header.condition.id',
+                defaultMessage: 'Condition ID',
+            }),
+            options: {
+                filter: true,
+                sort: true,
+            },
+        },
+        {
+            name: 'conditionType',
+            label: intl.formatMessage({
+                id: 'Admin.Throttling.Blacklist.Throttling.policy.table.header.condition.type',
+                defaultMessage: 'Condition Type',
+            }),
+            options: {
+                filter: true,
+                sort: false,
+            },
+        },
+        // TODO : Condition Status
+    ];
+
+    const emptyBoxProps = {
+        content: (
+            <Typography variant='body2' color='textSecondary' component='p'>
+                <FormattedMessage
+                    id='Throttling.Blacklist.Policy.List.empty.content.blacklist.policies and abuse by'
+                    defaultMessage={'By blacklisting requests, you can protect servers from common attacks'
+                    + ' users.'}
+                />
+            </Typography>),
+        title: (
+            <Typography gutterBottom variant='h5' component='h2'>
+                <FormattedMessage
+                    id='Throttling.Blacklist.Policy.List.empty.title.blacklist.policies'
+                    defaultMessage='Blacklist Policies'
+                />
+
+            </Typography>),
+    };
+
+    /**
+ * Mock API call
+ * @returns {Promise}.
+ */
+    function apiCall() {
+        let policyList;
+        let a = 0;
```

Comment:
Can we rename it to meaningful key instead of a?

Reply:
fixed